libs/core/langchain_core/callbacks/file.py

```diff
@@ -53,11 +53,10 @@ def on_chain_start(
         """
         if "name" in kwargs:
             name = kwargs["name"]
+        elif serialized:
```

Comment:
this is one of the few refactors i never liked: personally I find this easier to read while being more difficult to understand

Reply:
We could exclude this rule ?